lib/common/Mock.spec.ts

```diff
@@ -155,12 +155,12 @@ describe('Mock prototype', () => {
 
   it('should get all things mocked and in the same time respect prototype', () => {
     const mockDef = MockComponent(CustomComponent);
-    const mock = new mockDef();
+    const mock = new mockDef() as any;
     expect(mock).toEqual(jasmine.any(CustomComponent));
 
     // checking that it was processed through Mock
-    expect(mock.__ngMocksMock as any).toBe(true);
-    expect(mock.__ngMocksMockControlValueAccessor as any).toBe(true);
```

Comment:
unfortunately, it won't work like that and should be properly fixed to support TS 3.9. Due to required number of changes in architecture I have to overtake this task.